settings/ci.yml

```diff
@@ -33,6 +33,9 @@ sidekiq_redis_url: redis://127.0.0.1:6379/2
 efs_submissions_mount_point: "/tmp/exercism/efs/submissions"
 efs_repositories_mount_point: "/tmp/exercism/efs/repos"
 
+# GitHub Config
+github_organization:
```

Comment:
We don't necessarily need the github organization in CI, as all the tests should stub the organization anyway